ppr-ui/src/App.vue

```diff
@@ -14,20 +14,11 @@
       @proceed="payErrorDialogHandler()"
     />
 
-    <!-- Initial Page Load Transition -->
-    <transition name="fade">
-      <div class="loading-container" v-show="!haveData">
-        <div class="loading__content">
-          <v-progress-circular color="primary" size="50" indeterminate />
-          <div class="loading-msg">Loading</div>
-        </div>
-      </div>
-    </transition>
```

Comment:
If there is no more "spinner" in use anywhere in this app then its CSS (possibly in base) can be removed.